### ISSUE TITLE
Exception request - Experimental articulation support

### ISSUE THREAD

**Issue by hultonha** [o3de/o3de#15375]
Exception Requested to merge to stabilization by @hultonha (tagging @adamdbrw as secondary)

### Issues 

- [x] https://github.com/o3de/o3de/issues/15214
- [x] https://github.com/o3de/o3de/issues/15224

## Incremental PRs

- [x] https://github.com/aws-lumberyard-dev/o3de/pull/503
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/506
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/505
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/508
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/516
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/517
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/515
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/507
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/518
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/520
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/519
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/524
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/523
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/522
- [x] https://github.com/aws-lumberyard-dev/o3de/pull/521

### Final PR

- [x] https://github.com/o3de/o3de/pull/15494

## Exception Tracking

- [x] 1. Owning SIG requests the exception, by a Sig Representative (Chair/co-chair) adding the PR to the exception requests queue board.
- [x] 2. (@lgleim, SIG-Simulation) Owning SIG comments on the PR explaining impact if this code does not go into stabilization.
- [x] 3. (@lsemp3d, SIG-Content) Unrelated SIG representative (chair/cochair) approves the exception, adding a comment indicating approval and the SIG they represent.
- [x] 4. UX SIG representative (chair/cochair) approves the exception, adding a comment indicating approval and the SIG they represent.
- [x] 5. Testing SIG representative (chair/cochair) approves the exception, adding a comment indicating approval and the SIG they represent.
- [x] (Must occur at some point before SIG-Release approval) Code has been approved for merging by maintainers
- [x] 6. SIG-Release Release Manager or co-Release Manager approves the exception, adding a comment indicating approval and the SIG they represent.
- [x] 7. If all approvals are accepted, then the exception PR may be merged
- [x] 8. Merged PR is verified successful via testing. If Merged PR verification testing failed, consult reps from step 3-5 and discussion occurs on fixing the failure vs backing out change
- [x] 9. Exception is complete

**Comment by lsemp3d** [comment]
As chair of SIG-Content I approve this change, it brings a valuable feature but also several important improvements, fixes, and, authoring options for physics

**Comment by AMZN-Dk** [comment]
As a co-chair of SIG-Testing I approve of this change, the overall changes do not look to represent a large risk or impact to other systems. This approval is contingent on not merging today (3/31) as it is a Friday, but rather waiting until next week to merge to avoid the weekend.

**Comment by lgleim** [comment]
> Posting on behalf of @lgleim who is out on PTO - I take responsibility for this and hope Lars can confirm on his return.
> 
> > As chair of sig-simulation I would like to see this initial experimental support for articulations added to O3DE. If this is not merged then the robotics community will be missing out on a very important feature for the upcoming release.
> 
> (attrib @hultonha, based on prior conversations with Lars and @hultonha having been an active member of sig-simulation)

Thanks @hultonha! That is my stance indeed

**Comment by tonybalandiuk** [comment]
@hultonha  are we still targeting to get this in? If so, let's do it. Looks like only a couple of reviews to go!

**Comment by adamdbrw** [comment]
@Ulrick28 indeed the PR is ready to merge and as I understand the time window is closing tomorrow, could we make sure all the checkboxes are completed today? I will be responsive late if there are any questions or if anything needs confirmation.

**Comment by Ulrick28** [comment]
I will work to get the approvals

On Apr 11, 2023, at 2:04 AM, Adam Dąbrowski ***@***.***> wrote:

﻿

@Ulrick28<https://github.com/Ulrick28> indeed the PR is ready to merge and as I understand the time window is closing tomorrow, could we make sure all the checkboxes are completed today? I will be responsive late if there are any questions or if anything needs confirmation.

**Comment by Ulrick28** [comment]
Just need UX Sig, however not sure why we need UX sig anymore, as that was a requirement for RTE during the last release.

**Comment by moraaar** [comment]
Thanks for the approval. I'll merge first thing tomorrow morning (UK time) so I can keep an eye on AR.

**Comment by rainbj** [comment]
Sig-UX-UI approval. Thanks.

**Comment by moraaar** [comment]
PR has been merged into `stabilization/2305` branch and [AR](https://jenkins.build.o3de.org/blue/organizations/jenkins/O3DE/detail/stabilization%2F2305/90/pipeline/918/) has passed green.

**Comment by michalpelka** [comment]
I've done a sanity check on o3de/stabilization_2305 against mp/articulations_actuation. The feature is working as expected.
